modules/ROOT/pages/index.adoc

```diff
@@ -1,27 +1,13 @@
-= Bonita Intelligent Continuous Improvement Add-on
-:page-aliases: release_notes.adoc
+= Labs Program
 
-Monitor the duration of open cases, get help from an Artificial Intelligence algorithm providing the chances for each case to finish within a target duration, get all data to understand the lifecycle of a case, and follow the trends on the longer term thanks to the Analytics dashboards.
+image::images/Lab_icon.png[Labs logo,200,100]
```

Comment:
ℹ️ we could move the images directly in the images root folder as this is a new one.
This hadn't been done for existing pages because we didn't want to pass on all existing pages and changes the image links.
Let's see this when we talk about the Antora modules